packages/integration-tests/fixtures/deterministic-debug-ids/deterministic-debug-ids.test.ts

```diff
@@ -0,0 +1,110 @@
+/* eslint-disable jest/no-standalone-expect */
+/* eslint-disable jest/expect-expect */
+import childProcess from "child_process";
+import path from "path";
+import fs from "fs/promises";
+
+// eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+const nodejsMajorversion = process.version.split(".")[0]!.slice(1);
+
+function executeAndGetDebugIds(bundlePath: string): string[] {
+  const processOutput = childProcess.execSync(`node ${bundlePath}`, { encoding: "utf-8" });
+  const debugIdMap = JSON.parse(processOutput) as Record<string, string>;
+  return Object.values(debugIdMap);
+}
+
+function build(): void {
+  childProcess.execSync(`yarn ts-node ${path.join(__dirname, "build.ts")}`, { stdio: "inherit" });
+}
+
+beforeAll(async () => {
+  await fs.writeFile(
+    path.join(__dirname, "input", "dynamic-variable.js"),
+    `global.dynamicVariable = 1;`
+  );
+});
+
+afterAll(async () => {
+  await fs.unlink(path.join(__dirname, "input", "dynamic-variable.js"));
+});
+
+test("Same debug IDs for multiple identical builds", () => {
```

Comment:
l: This should either be a table test, or separate tests for rollup/vite/webpack.

Makes it easier to add new tests in future, and to instantly identify which bundler is broken